packages/js/src/dashboard/store/alert-center.js

```diff
@@ -0,0 +1,116 @@
+/* global ajaxurl */
+import { createSlice, createSelector } from "@reduxjs/toolkit";
+import { get } from "lodash";
+import { ASYNC_ACTION_NAMES, ASYNC_ACTION_STATUS } from "../../shared-admin/constants";
+
+export const ALERT_CENTER_NAME = "alertCenter";
+
+const TOGGLE_ALERT_VISIBILITY = "toggleAlertVisibility";
+
+/**
+ * @param {string} id The id of the alert.
+ * @param {string} nonce The nonce of the alert.
+ * @param {boolean} hidden The hidden state of the alert.
+ * @returns {Object} Success or error action object.
+ */
+export function* toggleAlertStatus( id, nonce, hidden = false ) {
+	yield{ type: `${ TOGGLE_ALERT_VISIBILITY }/${ ASYNC_ACTION_NAMES.request }` };
+	try {
+		yield{ type: TOGGLE_ALERT_VISIBILITY,
+			payload: {
+				id,
+				nonce,
+				hidden,
+			},
+		    };
+		return { type: `${ TOGGLE_ALERT_VISIBILITY }/${ ASYNC_ACTION_NAMES.success }`, payload: { id } };
+	} catch ( error ) {
+		return { type: `${ TOGGLE_ALERT_VISIBILITY }/${ ASYNC_ACTION_NAMES.error }`, payload: error };
+	}
+}
+
+/**
+ * Toggle alert visability.
+ *
+ * @param {object} state The state.
+ * @param {string} id The id of the alert to hide.
+ *
+ * @returns {void}
+ */
+const toggleAlert = ( state, id ) => {
+	state.alerts = state.alerts.map( ( alert ) => {
+		if ( alert.id === id ) {
+			return { ...alert, dismissed: ! alert.dismissed };
+		}
+		return alert;
+	} );
+};
+
+const slice = createSlice( {
+	name: ALERT_CENTER_NAME,
+	initialState: { alerts: [] },
+	reducers: {
+		toggleAlert,
+	},
+	extraReducers: ( builder ) => {
+		builder.addCase( `${ TOGGLE_ALERT_VISIBILITY }/${ ASYNC_ACTION_STATUS.success }`, ( state, { payload: { id } } ) => {
+			slice.caseReducers.toggleAlert( state, id );
+		} );
+	},
+} );
+
+/**
+ * @returns {Object} The initial state.
+ */
+export const createInitialAlertCenterState = slice.getInitialState;
```

Comment:
In the `shared-admin` we use `get` instead of `create` due to this toolkit naming